entgql/internal/todo/ent.graphql

```diff
@@ -712,7 +712,9 @@ type Todo implements Node {
   init: Map
   custom: [Custom!]
   customp: [Custom]
+  """The parent of the todo"""
   parent: Todo
+  """The children of the todo"""
```

Comment:
this comment and the comment for groups were not displayed before, even though the parent comment above was displayed.